### DISCUSSION TITLE
Is it possible to change text highlighting colors?

### DISCUSSION THREAD

**Discussion by ghjardim** [TriliumNext/Trilium#2024] [Q&A] [answered]
Hello. I'm wondering if it is possible to change the color palette used when highlighting text on Trillium. See:

![trilium-colors](https://user-images.githubusercontent.com/49923210/122624935-e4e81500-d078-11eb-9cab-14c3cdee5429.jpg)

Is it possible? How can it be done? It needs JavaScript or can I edit it through my custom CSS?

**Comment by zadam** [comment, marked as answer]
Hi, this is currently hardcoded in the CKEditor build and changing the color palette is currently not supported.
